files/fr/web/api/css_font_loading_api/index.html

```diff
@@ -0,0 +1,48 @@
+---
+title: CSS Font Loading API
+slug: Web/API/CSS_Font_Loading_API
+tags:
+  - API
+  - CSSFontLoading
+  - CSSOM
+  - Experimental
+  - Fonts
+  - NeedsContent
+  - Reference
+translation_of: Web/API/CSS_Font_Loading_API
+---
+<div>{{DefaultAPISidebar("CSS Font Loading API")}}{{SeeCompatTable}}</div>
+
+<p class="summary">L’API CSS Font Loading API fournit des évènements et interfaces pour le chargement dynamique des ressources associées aux polices.</p>
+
+<div class="notecard note">
+<p class="summary"><strong>Note :</strong> cette fonctionnalité est disponible dans <a href="/fr/docs/Web/API/Web_Workers_API">l’API Web Workers</a> (<code>self.fonts</code> donne accès à {{domxref('FontFaceSet')}}).</p>
```

Comment:
the `summary` class should not be included here again. (Here again this is a bug from the English content :s )